kibbeh/src/ui/ProfileHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactChild } from "react";
+import { ProfileHeaderWrapper } from "./ProfileHeaderWrapper";
+import { Button } from "./Button";
+import { UserBadge } from "./UserBadge";
+import { SingleUser } from "./UserAvatar/SingleUser";
+import { SolidMessages, SolidPersonAdd } from "../icons";
+
+import profileCover from "../stories/img/profile-cover.png";
+import src from "../img/avatar.png";
+
+export interface ProfileHeaderProps {
+  displayName: string;
+  username: string;
+  isFollowing?: boolean;
+  isOnline?: boolean;
+  children?: ReactChild;
+}
+
+export const ProfileHeader: React.FC<ProfileHeaderProps> = ({
+  displayName,
+  username,
+  isFollowing,
+  isOnline,
+  children,
+}) => {
+  return (
+    <ProfileHeaderWrapper coverUrl={profileCover}>
+      <div className="flex mr-4">
+        <SingleUser
```

Comment:
Profile picture is missing the outer border (outline). You can try with something like `outline: 4px solid var(--color-primary-800)`

Reply:
cool. I knew I was missing something lol. I'll make that quick fix

Reply:
You need to change it to outline for it to be an **outer** border. We don't want the profile picture getting smaller because of it